manifests/s/squidowl/halloy/2024.6/squidowl.halloy.installer.yaml

```
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: squidowl.halloy
+PackageVersion: 2024.6
```

```suggestion
PackageVersion: 0.0.2024.6
```
Please update file name and path to match this change.